hub/apps/how-tos/uno-multiplatform.md

```diff
@@ -45,75 +45,101 @@ Now that you've created your first WinUI 3 app, you might be wondering how to re
 
 4. Follow the instructions indicated by the tool.
```

Comment:
Make a note here that because uno-check needs to modify the system, it will request elevated permissions